gapir/cc/server_connection.h

```diff
@@ -56,11 +56,16 @@ class ServerConnection {
     // the posting was successful false otherwise.
     bool post(const void* postData, uint32_t postSize) const;
 
+    // Post a crash minidump from the given address with the given filename to the server. Returns
+    // true if the posting was successful false otherwise.
+    bool crash(const std::string& filename, const void* crashData, uint32_t crashSize) const;
```

Comment:
maybe rename `postCrashdump`? The other methods are verbs, and this doesn't sound like a function you'd want to call. 😄